doc/lua_api.txt

```diff
@@ -3664,7 +3664,9 @@ Definition tables
         ^ Don't forget to use "leveled" type nodebox. ]]
         liquid_range = 8, -- number of flowing nodes around source (max. 8)
         drowning = 0, -- Player will take this amount of damage if no bubbles are left
-        light_source = 0, -- Amount of light emitted by node (max. 14)
+        light_source = 0, --[[
+        ^ Amount of light emitted by node (max. 14).
+        ^ A value of 15 causes a lighting bug. ]]
```

Comment:
I considered that, but decided it was clearer to modders to mention the actual current values, as they will probably not be familiar with these constants as set in core code.